frontend/pages/create-project.tsx

```diff
@@ -67,9 +61,55 @@ const CreateProject: React.FC = () => {
         formik.setFieldValue('material', formik.values.material.filter((_, i) => i !== index));
     };
 
+    const formik = useFormik({
+        initialValues: {
+            projectName: '',
+            aboutProject: '',
+            selectedFile: null,
+            listOfParticipant: [] as string[],
+            newList: '',
+            material: [] as string[],
+            newMaterial: '',
+        },
+        onSubmit: async (values) => {
+            if (!formik.isValid || !formik.values.projectName.trim() || !formik.values.aboutProject.trim()) {
+                if (!formik.values.projectName.trim()) {
+                    setErrorMessage('The Project Name field is necessary !');
+                } else if (!formik.values.aboutProject.trim()) {
+                    setErrorMessage('The About Project field is necessary !');
+                }
+                return;
+            }
+            try {
+                await fetch('http://localhost:3000/projects', {
+                    method: 'POST',
+                    headers: {
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify({
+                        name: values.projectName,
+                        description: values.aboutProject,
+                        image: values.selectedFile,
+                        member_id: values.listOfParticipant,
+                        leader_id: '111',
+                        status: 'pending',
+                    }),
+                });
+                formik.resetForm();
+                setErrorMessage('');
+                alert('Your project has been created !');
```

Comment:
Use a toaster instead of an alert.
Take a look at react-hot-toast or react-toastify